### ISSUE TITLE
error: 'is_latex_output' is not an exported object from 'namespace:knitr'

### ISSUE THREAD

**Issue by shaolintl** [haozhu233/kableExtra#216]
Hello,

I am getting the following error:
```
Error : .onLoad failed in loadNamespace() for 'kableExtra', details:
  call: NULL
  error: 'is_latex_output' is not an exported object from 'namespace:knitr'
Error: package or namespace load failed for 'kableExtra'
```

Versions are:
```
R version 3.3.3 (2017-03-06)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 9 (stretch)
knitr_1.20.3
rmarkdown_1.9.18
```

Thank you.

**Comment by haozhu233** [comment]
Hi, @shaolintl Could you try to update your knitr to latest CRAN version? I think it will solve this problem.

**Comment by LunaSare** [comment]
I was getting the same error, and it was solved with your suggestion @haozhu233 , thanks!

**Comment by shaolintl** [comment]
Hi @haozhu233, Thank you for your reply. I tried using knitr stable (1.20) but the problem seemed to be resolved only when I upgraded R to 3.4.

Thank you again!
